src/services/account.ts

```diff
@@ -0,0 +1,429 @@
+}
+
+export class Account extends Service {
+
+    /**
+     * Get Account
+     *
+     * Get currently logged in user data as JSON object.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async get(): Promise<string> {
+        let path = '/account';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * Create Account
+     *
+     * Use this endpoint to allow a new user to register a new account in your
+     * project. After the user registration completes successfully, you can use
+     * the [/account/verfication](/docs/account#createVerification) route to start
+     * verifying the user email address. To allow your new user to login to his
+     * new account, you need to create a new [account
+     * session](/docs/account#createSession).
+     *
+     * @param string email
+     * @param string password
+     * @param string name
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async create(email: string, password: string, name: string = ''): Promise<string> {
+        let path = '/account';
+        
+        return await this.client.call('post', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'email': email,
+                'password': password,
+                'name': name
+            });
+    }
+
+    /**
+     * Delete Account
+     *
+     * Delete a currently logged in user account. Behind the scene, the user
+     * record is not deleted but permanently blocked from any access. This is done
+     * to avoid deleted accounts being overtaken by new users with the same email
+     * address. Any user-related resources like documents or storage files should
+     * be deleted separately.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async delete(): Promise<string> {
+        let path = '/account';
+        
+        return await this.client.call('delete', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * Update Account Email
+     *
+     * Update currently logged in user account email address. After changing user
+     * address, user confirmation status is being reset and a new confirmation
+     * mail is sent. For security measures, user password is required to complete
+     * this request.
+     *
+     * @param string email
+     * @param string password
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async updateEmail(email: string, password: string): Promise<string> {
+        let path = '/account/email';
+        
+        return await this.client.call('patch', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'email': email,
```

Comment:
when your values having the same names as the keys you can do:
`{email, password}`